docs/faq/index.mdx

```diff
@@ -0,0 +1,59 @@
+# FAQ
+
+Here is a list of questions that have been previously posed by various members.
+Please review them carefully as they might address some of the concerns or inquiries you might have.
+
+<Accordion title="How to configure the widgetbook builder in build.yaml">
+You can customize the builder as follows and you can find more configuration in the `build_config` package.
+
+```yaml
+targets:
+  $default:
+    builders:
+      widgetbook_generator:use_case_builder:
+        generate_for:
+          - lib/widgets/**
+```
+
+</Accordion>
+
+<Accordion title="How to access the custom theme?">
+	For more detailed information, visit the [Custom Theme in
+	Widgetbook](/cookbook/custom-theme).
+</Accordion>
+
+<Accordion title="How would I inject a SafeArea or other widgets into the Device Frame for all use cases?">
+	All devices have a SafeArea by default. Please submit a GitHub issue with the
+	bottom sheet issue (including a reproduction code) to be able to track it
+	better there.
+</Accordion>
+
+<Accordion title="How do we create a WidgetBook account since the 'Get Access' button redirects to Discord without a signup option?">
```

Comment:
```suggestion
<Accordion title="How do we create a Widgetbook account since the 'Get Access' button redirects to Discord without a signup option?">
```